javascript/nitro/nitro.config.ts

```diff
@@ -0,0 +1,10 @@
+//https://nitro.unjs.io/config
+export default defineNitroConfig({
+    // File with [] in name can't be copied properly
```

Comment:
Btw @waghanza, I am not very familiar with how Ruby works, would you happen to be able to know a solution for this issue?

Essentially, the original file is `routes/user/[id].get.ts`, however, when the Dockerfiles are generated, I am unable to add proper escaping to the file.

I believe the correct Dockerfile COPY command that should be used is:

`COPY 'routes/user/[[]id].get.ts' 'routes/user/[[]id].get.ts'`

But the generated Dockerfile is:

`COPY 'routes/user/[id].get.ts' 'routes/user/[id].get.ts'`

Thank you!! 🙏🏻 

Docker docs ref: https://docs.docker.com/engine/reference/builder/#copy

Reply:
Honestly, I could find a solution for this, but it seems very awkward for me to have some square brackets in filenames. 

It tend to keep using [a-zA-Z0-9] for filenames

Reply:
I think we can resolve this if we only intend to keep the `h3` benchmarks